cypress/e2e/shared/annotations.range.test.ts

```diff
@@ -0,0 +1,258 @@
+import {
```

Comment:
I like this naming convention with the `.` separator better than what we've been doing...but I am concerned that it might not work with the IDPE CI. There is some fancy regex / grep stuff happening over there to self-discover these files and I'm not sure how flexible it is. Might want to ask @wojciechka

Reply:
Checked this and it should be fine after some small changes. Let's wait for my changes to support this in all of our CI pipelines before merging this, though. Should be done in a bit.